pyvcloud/vcd/org.py

```diff
@@ -582,6 +608,39 @@ def get_roles_query(self, name_filter=None):
             qfilter=org_filter)
         return query, resource_type
 
+    def get_right(self, right_name):
+        """
+        Retrieves corresponding record of the specified right.
+        :param right_name: (str): The name of the right record to be retrieved
+        :return: (QueryResultRightRecordType): Right query result in records
+                 format
+        """  # NOQA
+        try:
+            rights_query = self.get_rights_query(('name', right_name))[0]
+            return rights_query.find_unique()
+        except MissingRecordException:
+            raise Exception('Right \'%s\' does not exist.' % right_name)
+
+    def get_rights_query(self, name_filter=None):
```

Comment:
This looks out of place. Query related stuff should be in client.py

Reply:
We have `list_*`  methods outside client.py. Maybe this should renamed to `list_rights`?